pkg/remote/builtin/github/github.go

```diff
@@ -34,12 +37,14 @@ type GitHub struct {
 
 func New(conf *config.Config) *GitHub {
```

Comment:
we're actually getting rid of the config file all together and moving to a driver-based model. There is a version of this implemented in the database:
https://github.com/drone/drone/blob/0.4-database/pkg/store/store.go#L16

We should do something like `remote.Load(os.Getenv("REMOTE_DRIVER))` which should find the appropriate driver and return a valid remote connection. The remote connection should retrieve its data from environment variables defined here:
http://readme.drone.io/docs/setup/#/github.md

Let me know if that makes sense :)